packages/functional-tests/playwright.config.ts

```diff
@@ -54,7 +55,7 @@ const config: PlaywrightTestConfig<TestOptions, WorkerOptions> = {
       testMatch: 'stub.spec.ts',
       use: {
         browserName: 'firefox',
-        targetName: 'local',
+        //targetName: 'local',
```

Comment:
Q: why was this commented out?

Reply:
➕ I'm wondering if we could remove stub.spec.ts and this part of the configuration all together? 

From the [README](https://github.com/mozilla/fxa/tree/main/packages/functional-tests#creating-new-tests) my understanding is that it's used when creating new tests, but in a suite of 300+ do we really still need this?